packages/wrangler/src/__tests__/dev.test.ts

```diff
@@ -1498,6 +1498,9 @@ describe.sequential("wrangler dev", () => {
 				env.NAME_3 (CLASS_3)                           Durable Object      local
 				env.NAME_4 (CLASS_4, defined in SCRIPT_B)      Durable Object      local [not connected]
 
+
+				Service bindings, Durable Object bindings, and Tail consumers connect to other Wrangler or Vite dev processes running locally, with their connection status indicated by [connected] or [not connected]. For more details, refer to https://developers.cloudflare.com/workers/runtime-apis/bindings/service-bindings/#local-development
```

Comment:
```suggestion
				Service bindings, Durable Object bindings, and Tail consumers connect to other Wrangler or Vite processes running locally, with their connection status indicated by [connected] or [not connected]. For more details, refer to https://developers.cloudflare.com/workers/runtime-apis/bindings/service-bindings/#local-development
```

Reply:
I will update it in the follow up PR with all other message changes (as they impact a huge amount of snapshot tests in wrangler).